blog/2024-12-01-StartUpWeekExperience.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Saint Louis Startup Connection: Showcasing Open Source SLU’s Value to the Innovation Ecosystem
+description: A first-hand account of representing Open Source SLU at the Saint Louis Startup Connection, where I discovered a world of innovation, collaboration, and entrepreneurial energy.
+slug: blog_a_night_at_stl_startup_connection
+authors: [Chirag Gupta]
+tags: [announcement, event, entrepreneurship, open source]
+hide_table_of_contents: false
+---
+
+
+## Saint Louis Startup Connection: Showcasing Open Source SLU’s Value to the Innovation Ecosystem
+
+![Cover Photo of the Saint Louis Startup Connection Event](RepresentingOpenSourceInStartUpWeek/IMG_7220.jpg "Cover photo of the Saint Louis Startup Connection Event")
+
+The **Saint Louis Startup Connection** ([official website](https://www.stlouisstartupweek.com)) continues to be a cornerstone of the region’s entrepreneurial ecosystem, where founders, investors, and innovators converge to share ideas and spark collaborations. This year, **Open Source SLU (OSS-SLU)**—our experiential software engineering program at Saint Louis University—joined the lineup to showcase how we contribute to the growth and success of early-stage ventures.
```

Comment:
The parenthetical "official website" is awkward, please move the link to be on "Saint Louis Startup Connection" directly.

Reply:
Our organization is "Open Source with SLU". The "with" needs to be in the name every time it's mentioned.